libobs/audio-monitoring/pulse/pulseaudio-output.c

```diff
@@ -4,6 +4,8 @@
 #define PULSE_DATA(voidptr) struct audio_monitor *data = voidptr;
 #define blog(level, msg, ...) blog(level, "pulse-am: " msg, ##__VA_ARGS__)
 
+volatile long audio_track_active;
```

Comment:
This variable should be static.
```suggestion
static volatile long audio_track_active;
```